### PR TITLE
Cluster related enhancements

### PR DESCRIPTION
**Pull request by dougm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 ### (unreleased)
 
+* Add object.HostSystem methods: EnterMaintenanceMode, EnterMaintenanceMode, Disconnect, Reconnect
```
Comment (pietern) [inline CHANGELOG.md]: 2x EnterMaintenanceMode

File: govc/cluster/change.go [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+	// DRS
+	f.Var(flags.NewOptionalBool(&cmd.DrsConfig.Enabled), "drs-enabled", "Enable DRS")
+
+	drsModes := []string{
+		string(types.DrsBehaviorManual),
+		string(types.DrsBehaviorPartiallyAutomated),
+		string(types.DrsBehaviorFullyAutomated),
+	}
+	f.StringVar((*string)(&cmd.DrsConfig.DefaultVmBehavior), "drs-mode", "",
+		"DRS behavior for virtual machines"+strings.Join(drsModes, ", "))
```
Comment (pietern) [inline govc/cluster/change.go]: Add ": " or something before strings.Join?

File: govc/cluster/change.go [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+	}
+
+	for _, path := range f.Args() {
+		clusters, err := finder.ClusterComputeResourceList(ctx, path)
+		if err != nil {
+			return err
+		}
+
+		for _, cluster := range clusters {
+			fmt.Println(cluster.InventoryPath)
```
Comment (pietern) [inline govc/cluster/change.go]: Debug or intentional?

File: govc/host/destroy.go [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+	*flags.HostSystemFlag
+}
+
+func init() {
+	cli.Register("host.destroy", &destroy{})
+}
+
+func (cmd *destroy) Register(f *flag.FlagSet) {}
+
+func (cmd *destroy) Process() error { return nil }
```
Comment (pietern) [inline govc/host/destroy.go]: So this removes a host from a cluster? Could you add a little usage doc?

**Comment by pietern** [comment]
Some minor comments, LGTM otherwise

**Comment by dougm** [comment]
Thanks for the review @pietern - addressed all comments.  btw I'm tempted to rename host.destroy to host.remove, but stuck with 'destroy' as we use that name for govc calls to Destroy_Task.  Let me know if you'd prefer host.remove instead.

**Comment by pietern** [comment]
@dougm Agree `host.remove` is better, since the host won't be destroyed at all ;)

Also, it's symmetric with `host.add`.

**Comment by dougm** [comment]
@pietern host.remove it is

**Comment by pietern** [comment]
LGTM! :+1:

**Pull request merged**